tests/foreman/cli/test_discoveredhost.py

```diff
@@ -73,6 +75,11 @@ def setUpClass(cls):
 
         # Build PXE default template to get default PXE file
         Template.build_pxe_default()
+        # let's just modify the timeouts to speed things up
+        ssh.command("sed -i 's/TIMEOUT [[:digit:]]\+/TIMEOUT 1/g'"
+                    "/var/lib/tftpboot/pxelinux.cfg/default")
+        ssh.command("sed -e '/APPEND initrd/s/$/ fdi.countdown=1/'"
+                    "/var/lib/tftpboot/pxelinux.cfg/default")
```

Comment:
this speeds the boot process considerably without a need of manipulating the templates (which would require either unlocking or cloning)